File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js

//服务
app.service('brandService',function($http){
			//查询方法findAll;
			this.findAll = function(){
				return $http.get('../brand/findAll.do');
			}
			
			//分页查询findPage
			this.findPage=function(currentPage,pageSize){
				return $http.get("../brand/findPage.do?currentPage="+currentPage+"&pageSize="+pageSize);
			}
			
			//add
			this.add=function(entity){
				return $http.post("../brand/add.do",entity);
			}
			//findOne
			this.findOne=function(id){
				return $http.get("../brand/findOne.do?id="+id);
			}
			//update
			this.update = function(entity){
				return $http.post("../brand/update.do",entity);
			}
			//dele
			this.dele = function(ids){
				return $http.post("../brand/delete.do?ids="+ids);
			}
			//search
			this.search = function(currentPage,pageSize,entity){
				return $http.post("../brand/search.do?currentPage="+currentPage+"&pageSize="+pageSize,entity);
			}
			
			//selectOptionList
			this.selectOptionList = function(){
				return $http.get("../brand/SelectOptionList.do");
			}
		}); 

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

public interface BrandService {
	
	public List<TbBrand> findAll();
	
	//public PageResult findPage(int pageNo,int pageSize);
	
	public void add(TbBrand tbBrand);
	
	public TbBrand findOne(Long id);
	
	public void update(TbBrand tbBrand);
	
	public void delete(Long [] ids);
	
	public PageResult findPage(int pageNo,int pageSize,TbBrand tbBrand);
	
	public List<Map> selectOptionList();
}
